database/index.js

```diff
@@ -114,7 +114,7 @@ const getOneProdLoc = (id) => {
           console.log(error);
           reject(error);
         } else {
-          // console.log(result);
+          // console.log('console.log from get1ProdLoc 😋', result);
```

Comment:
Once you've tested your functionality on the database, its probably best to get rid of all the console logs